security-insights-schema-1.0.0.yaml

```diff
@@ -130,6 +130,35 @@ properties:
                 $id: '#/properties/contribution-policy/properties/accepts-bot-pull-requests'
                 description: 'Define if the maintainers accept pull-requests generated by bots or automated tools.'
                 type: boolean
+            bots-list:
+                $id: '#/properties/contribution-policy/properties/bots-list'
+                description: 'List of allowed and denied bots. This property can overwrite `accepts-bot-pull-requests`.'
+                additionalItems: false
+                items:
+                    anyOf:
+                    -   $id: '#/properties/contribution-policy/properties/bots-list/items'
+                        additionalProperties: false
+                        properties:
+                            bot-name:
+                                $id: '#/properties/contribution-policy/items/anyOf/0/properties/bot-name'
+                                description: 'Bot name.'
+                                type: string
+                            action:
+                                $id: '#/properties/contribution-policy/items/anyOf/0/properties/action'
+                                description: 'Define if bot actions are allowed or denied.'
+                                type: string
+                                enum: ['allowed', 'denied']
+                            comment:
+                                $id: '#/properties/contribution-policy/items/anyOf/0/properties/comment'
+                                description: 'Short comment.'
+                                type: string
+                                pattern: '^(.|\n){1,560}$'
+                        required:
+                        - bot-name
+                        - action
```

Comment:
Should a comment be required?

Reply:
In the spec all the comments are optional, I don't think it is so important for a scanner, but it can helpful for humans, but humans can open an issue and ask the maintainers the reasons.

Reply:
> I don't think it is so important for a scanner

As a tool potentially implementing a scanner to pull these files down, I would want to know why my automation had been rejected by the maintainer. I would collect all `comments` and put them into a report

Reply:
Any further thoughts here?